docs/content/_index.es.md

```diff
@@ -11,6 +11,13 @@ date: 2019-03-14T08:44:12+00:00
 
 En esta página web, está documentado Athens en detalle. Te enseñaremos que hace, por que es necesario, y como puedes ejecutarlo por tu cuenta. A continuación te mostramos un breve listado.
 
+#### ¿Cómo empezar?
+    Correr `docker run -p '3000:3000' gomods/athens:latest`
+
+Luego, configura tu `GOPROXY` y `go get` go!
```

Comment:
Luego means later, I wouldn't use this one here.

Not sure about how to translate this word joke but...

"Ahora configura tu `GOPROXY` y a darle al `go get`!

Could work.